### ISSUE TITLE
EndGetStatelessServiceFactory found an entry but it does not have valid stateless service factory

### ISSUE THREAD

**Issue by jseekamp** [coherentsolutionsinc/aspnetcore-service-fabric-hosting#96]
Thank you so much for this project!

I feel like I must be doing something wrong here so please bear with me. I am tryin to deploy a very simple application to my local service fabric cluster as a proof of concept and I keep getting the following error in my Event Viewer. The application deploys to service fabric but it looks like the node just continuously recycles. The error that the node reports is the following:

Replica had multiple failures during open on _Node_0. -2147017774

Have you ever seen anything like this when using your library?

My sample project is here if you have time to take a look: 
https://github.com/jseekamp/ServiceFabric

**Comment by OlegKarasik** [comment]
Hi @jseekamp,

Thanks for a nice feedback. Unfortunately I won't have access to PC during the weekend, but I will check the issue on Monday.

**Comment by OlegKarasik** [comment]
Hello again,

I have created a [PR](https://github.com/jseekamp/ServiceFabric/pull/1) for the example repository you have provided.

Here are the explanations:

1. In [this](https://github.com/jseekamp/ServiceFabric/pull/1/commits/8d054a9cdbfd748640fc7d31fd4c3e09de365c94) commit I added a call to `Connect-ServiceFabricCluster` cmdlet to make sure these scripts will work even if you run them without prior establishing connection to the Cluster.
2. In [this](https://github.com/jseekamp/ServiceFabric/pull/1/commits/dda306ef3295d1da4136cd96f037a8f9192e5462) commit I have added a call to `app.UseRoutine()`. In .NET Core 3.x this should be made before any of the endpoints can be registered (because all endpoints are handled by the EndpointMiddleware registered by `UseRouting` call).
3. In [this](https://github.com/jseekamp/ServiceFabric/pull/1/commits/0276eb419a14fc6a80bcd5b4b550c1cd5e60b402) commit I have changes the registration from **Stateful** to **Stateless**, because according to manifest files `WebType` is a stateless service.

Besides the above changes it is important to ensure you have active Service Fabric cluster deployed and started (because sometimes the mentioned exception could be a result of an attempt to debug application which isn't deployed to Service Fabric cluster).
